engine/input/src/input.cpp

```diff
@@ -1201,7 +1211,19 @@ namespace dmInput
                 }
             }
         }
+    }
+
+    bool GamepadConnectivityCallback(uint32_t gamepad_index, bool connected, void* userdata)
+    {
+        Context* context = (Context*)userdata;
+        Binding* binding = (Binding*)context->m_Binding;
+        if (!binding)
+        {
+            context->m_UnmappedGamepads.Put(gamepad_index, connected);
+            return true; // Only return false if you wish to override the controller connection status!
```

Comment:
The input system is still a little bit of a mess.
SImply returning true here "solves" the issue, since the HID gamepad will then get connected, the SetupGamePads will create a binding for that gamepad, and the next UpdateActions will correctly map the content. But the code flow is a bit unintuitive.

So, I think getting a list of pending connections here is a good "documentation" strategy.